src/main/java/com/yugabyte/sample/apps/AppBase.java

```diff
@@ -209,14 +209,16 @@ protected Connection getPostgresConnection(String database, String username, Str
   public void initializeConnectionsAndStatements(int numThreads) { }
 
   protected static void createKeyspace(Session session, String ks) {
-    String create_keyspace_stmt = "CREATE KEYSPACE IF NOT EXISTS " + ks +
-      " WITH REPLICATION = {'class': 'SimpleStrategy', 'replication_factor' : 1};";
-    // Use consistency level ONE to allow cross DC requests.
-    session.execute(
-      session.prepare(create_keyspace_stmt)
-        .setConsistencyLevel(ConsistencyLevel.ONE)
-        .bind());
-    LOG.debug("Created a keyspace " + ks + " using query: [" + create_keyspace_stmt + "]");
+    if (!appConfig.skipDDL) {
```

Comment:
DDL covers not just keyspace creation.

What about createTablesIfNeeded() ?

If node is down, wouldn't DML be affected as well ?

Reply:
See changes above to Main.java